CHANGELOG.rst

```diff
@@ -1,6 +1,22 @@
 Changelog
 =========
 
+0.0.1.dev3 (2018-06-14)
+-------------------------
+Added
+~~~~~
+* Add IRunnable, IMessageHandler
+* Add route configuration requirement
+
+Changed
+~~~~~~~
+* Require IEventMessage to have `phase` and `msg_id`
```

Comment:
It's so annoying when going between markdown and rst, I'm sorry 😬 but single back ticks in rst means italics; code/\<pre\> stuff is wrapped in double ticks.